sentry-android-core/src/main/java/io/sentry/android/core/InternalSentrySdk.java

```diff
@@ -0,0 +1,96 @@
+package io.sentry.android.core;
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import io.sentry.DateUtils;
+import io.sentry.HubAdapter;
+import io.sentry.ILogger;
+import io.sentry.ObjectWriter;
+import io.sentry.Scope;
+import io.sentry.SentryLevel;
+import io.sentry.protocol.App;
+import io.sentry.protocol.Device;
+import io.sentry.protocol.User;
+import io.sentry.util.MapObjectWriter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@ApiStatus.Internal
```

Comment:
Maybe just add a comment/Javadoc for the reason of this class, since it's not used in the SDK directly? even something like "This is going to be used by hybrid SDKs to get the objects they need" should suffice